packages/core/review-workflows/admin/src/routes/settings/:id.tsx

```diff
@@ -113,15 +112,14 @@ const EditPage = () => {
   const { _unstableFormatValidationErrors: formatValidationErrors } = useAPIErrorHandler();
   const navigate = useNavigate();
   const { toggleNotification } = useNotification();
-  const dispatch = useDispatch();
   const {
     isLoading: isLoadingWorkflow,
     meta,
     workflows,
     error,
     update,
     create,
-  } = useReviewWorkflows({ id: isCreatingWorkflow ? undefined : id });
+  } = useReviewWorkflows();
```

Comment:
Now we've removed ID from here doesn't it make this whole route redundant?

https://github.com/strapi/strapi/blob/ee7979c6d5a886a44ba3e11aebb41c417f30caa7/packages/core/review-workflows/server/src/routes/review-workflows.ts#L75-L91

If in every case in the FE we need access to all workflows maybe we should
remove this route?